docs/GettingStartedDocs/GettingStarted.Windows.md

```diff
@@ -149,6 +159,11 @@ C:\openenclave\build\x64-Debug
 https://docs.microsoft.com/en-us/dotnet/framework/tools/developer-command-prompt-for-vs)
 Normally this is accessible under the `Visual Studio 2017` folder in the Start Menu.
 2. At the x64 Native Tools command prompt, use cmake and ninja to build the debug version:
+To build with DCAP libraries for Windows, add -DUSE_LIBSGX=ON to the cmake command below
+as the default build for Windows is still -DUSE_LIBSGX=OFF i.e. AESM flow.
+However, please ensure that your ACC VM is configured with the DCAP libraries for Windows.
+Current Limitation is that these libraries are installed in c:\openenclave\prereqs\nuget folder.
+Copy or move them to your openenclave directory.
```

Comment:
What does this mean? Why does it need to be in the openenclave root folder and not the nuget subfolder?

Reply:
The ACC VM that was deployed using oe-engine and ansible had the nuget libraries in c:\openenclave folder. I copied the nuget directory to my openenclave folder for the build with dcap to work.

Since ansible cannot be run on the ACC VM as we cannot install WSL on these machines, we need to point out this limitation. We should probably have a power-shell script that automates the DCAP library nuget package generation to resolve this.

Reply:
> The ACC VM that was deployed using oe-engine and ansible had the nuget libraries in c:\openenclave folder.

This sounds like a bug. One of the goals of updating this document is to ensure that the host CMakeLists.txt dependencies line up with the ansible configurations on Windows. Please work with Cloudbase to reconcile this difference.

Also, are you sure that your configuration is in sync with the latest sources? The [windows.yml](https://github.com/microsoft/openenclave/blob/04fdce70419f12912c194d472e89734a1604fec1/scripts/ansible/roles/windows/az-dcap-client/vars/windows.yml) indicates that the target path is C:\openenclave\prereqs\nuget, not in the root openenclave folder.

> Since ansible cannot be run on the ACC VM as we cannot install WSL on these machines, we need to point out this limitation. We should probably have a power-shell script that automates the DCAP library nuget package generation to resolve this.

That's a fine next step for this which we should work with CloudBase for; my requirement for this PR though is that we have a comprehensive set of instructions in this doc that can be translated easily into a script for automation.

Reply:
ok, sounds good.

Reply:
Removed this for now and will work with CloudBase to get this resolved.